README.md

```diff
@@ -2,62 +2,22 @@
 
 [KVIB](https://kartverket.github.io/kvib/storybook) er monorepoet til Kartverkets designsystem.
 
-## Innhold
+## Introduksjon
 
-Dette repoet inneholder:
-
-- `apps/storybook`: dokumentasjonssiden for designsystemet
-- `packages/react`: komponentbiblioteket i React
+https://kartverket.github.io/kvib/storybook?path=/docs/introduksjon--docs
 
 ## Dokumentasjonsside
 
-Dokumentasjonssiden for designsystemet finner du på https://kartverket.github.io/kvib/storybook. Der dokumenterer vi tokens, retningslinjer for bruk og alle komponentene.
+Dokumentasjonssiden for designsystemet finner du på https://kartverket.github.io/kvib/storybook. Der dokumenterer vi
+tokens, retningslinjer for bruk og alle komponentene.
 
 ## Ta i bruk
 
-For å ta i bruk komponentbiblioteket må du installere `@kvib/react`.
```

Comment:
kanskje beholde deler av denne for å beskrive linken. Slik som er gjort unedr "## Dokumentasjonsside"